### PR TITLE
Issue #969: fix the return documentation and word wrapping in the entity_load() doc block.

### PR DESCRIPTION
**Pull request by rszrama**
Note that I also fixed the word wrapping in the general function description. Not sure why it was wrapped so oddly, but maybe the class name used to be longer. : P

### CODE REVIEW

File: core/modules/entity/entity.module [hunk 1]
```diff
@@ -185,12 +185,11 @@ function entity_create_stub_entity($entity_type, $ids) {
  * database access if loaded again during the same page request.
  *
  * The actual loading is done through a class that has to implement the
- * EntityControllerInterface interface. By default,
- * DefaultEntityController is used. Entity types can specify that a
- * different class should be used by setting the 'controller class' key in
- * hook_entity_info(). These classes can either implement the
- * EntityControllerInterface interface, or, most commonly, extend the
- * DefaultEntityController class. See node_entity_info() and the
+ * EntityControllerInterface interface. By default,DefaultEntityController is
```
Comment (quicksketch) [inline core/modules/entity/entity.module]: Nitpick (but that's what we're doing here), there needs to be a space after "By default,"

Comment (rszrama) [inline core/modules/entity/entity.module]: D'oh. Gives me an opportunity to figure out how to update a pull request. :P

Comment (quicksketch) [inline core/modules/entity/entity.module]: Just checkout the same branch again, make the fix then:

```
git commit --amend
git push origin [branch_name] -f
```

Yes, GitHub supports force-updating PRs (even in a way encourages it). It's what drives Linus Torvalds mad about GitHub.

**Comment by rszrama** [comment]
Roger, roger. Fixed 'er up. Feel only a little dirty.

**Comment by rszrama** [comment]
Hmm, there's no such thing as pull request statuses, are there? It's just pass / fail on tests, but no "needs work" equivalent?

**Comment by quicksketch** [comment]
Yeah we mark them as needs work here in the issue instead of on the PR. You _can_ set labels on PRs, but as only users with commit access can set labels, we usually track them in the issue queue instead. In a way it's better as it mostly keeps the thread of conversation in the issue.

**Comment by quicksketch** [comment]
Thanks! Merged into 1.x and 1.1.x. Sometimes we get random failures from Travis, we haven't yet been able to track them down. Issue https://github.com/backdrop/backdrop-issues/issues/511

**Pull request merged**